microservices/submission-service/src/replay-validation/replay-validation.service.spec.ts

```diff
@@ -0,0 +1,754 @@
+import {CoreEndpoint, CoreService, MatchmakingService, MinioService, ResponseStatus} from "@sprocketbot/common";
+import {Readable} from "stream";
+import {anything, instance, mock, when} from "ts-mockito";
```

Comment:
NestJS generally sets up some pretty good boiler-plate for testing, including a testing module where you can inject mocked dependencies. Is there a reason you're using mockito instead of that?

https://docs.nestjs.com/fundamentals/testing#testing-utilities

Reply:
Fairly certain we had this discussion two months ago when it was relevant, but the short answer is I couldn't get Nest's boilerplate mocks to work very well, and mockito worked perfectly. I'm happy to revisit down the line, but I'm not planning to update this to go back to Nest/Jest in this PR.